tests/scripts/all.sh

```diff
@@ -1120,6 +1120,70 @@ component_test_cmake_out_of_source () {
     unset MBEDTLS_ROOT_DIR
 }
 
+component_test_basic_mbed_os_conf () {
+msg "build+test: Mbed OS default configuration"
+    scripts/config.pl unset MBEDTLS_NET_C
```

Comment:
Rather than call `config.pl` a bunch of times, how about defining the Mbed OS configuration in a file in `configs`? Or as a filter in `config.pl`, if we want to define it by comparison to the default configuration?